### ISSUE TITLE
meta description should vary by page, not only by website

### ISSUE THREAD

**Issue by Nick-Levinson** [h5bp/html5-boilerplate#1768]
I assumed the Description meta tag should be consistent across a given website. I forgot who disabused me of that notion. Consistency makes it look like a bad website. It should generally be unique for each unique page.

This should be added to extend.md > Search, with a new subsection, Description Meta Tag.

**Comment by alrra** [comment]
@Nick-Levinson Feel free to open a [pull request](https://github.com/h5bp/html5-boilerplate/blob/d35a8e902046bd77576b37039298e5206f9e3b90/CONTRIBUTING.md#pull-requests)!

**Comment by Nick-Levinson** [comment]
Thank you, but I found out for another issue that doing so would probably take a few hours to learn and carry out, assuming I have the software for it, and I'm moderately geeky. I'll leave this as a suggestion for anyone else to act on or not. Thanks again.

**Comment by roblarsen** [comment]
@Nick-Levinson For what it's worth, this would be a documentation PR, so you could theoretically just start editing the page in question right here on Github. Hit the fork button, edit the page, save it and then you can issue a pull request. All from GitHub. You actually have done the work here (researching all these topics) so it makes sense that you be the one to actually get credit for the work.

**Comment by Nick-Levinson** [comment]
I'll pass on doing a PR. I just tried. The method without installing Git required clicking on an icon that's not visible, at least under my login. To install Git, I opened the Software app on my openSuse 13.2 Linux laptop (probably a Gnome app) but it didn't list Git. I'm cautious about downloading something they don't list, as that might portend an incompatibility that might break something else on my laptop, and openSuse/Gnome already has peculiarities that I'm being patient about (I've reported them). I'm likely to get the PC-BSD OS for when openSuse reaches its end-of-life but that might be a couple of months. The Software app lists kdesvn, which I presume is in the same category, but I don't know if it's compatible with GitHub and, while gross lockouts are not much of a problem, subtle incompatibilities could be someone's headache and I don't want to be offering digital aspirin and apologies.

I'm not worried about credit. Should my ego want burnishing, the thread is good enough.

Thank you very kindly for the thought.
